src/plzinit/go.go

```diff
@@ -0,0 +1,67 @@
+package plzinit
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/thought-machine/please/src/cli"
+)
+
+// golangConfig prompts the user and returns the go config to add to the .plzconfig
+func golangConfig(dir string) string {
+	//TODO(jpoole): make this check that go is in the please path
+	//TODO(jpoole): detect if we're in a go project and suggest they enable GO_BUILD_COMPAT to help migration
+
+	goModule, ok := findGoModule(dir)
+
+	if !ok && !cli.PromptYN("Would you like to setup golang in this repository", false) {
+		return ""
+	}
+
+	config := "\n[go]\n"
+
+	goRoot, err := cli.Prompt("Do you want to set GOROOT? Otherwise please will use `go env GOROOT`", os.Getenv("GOROOT"))
+	if err != nil {
+		os.Exit(1)
+		return "" // most likely ctr-c
+	}
+	if goRoot != "" {
+		config += fmt.Sprintln("goroot = ", goRoot)
+	}
+
+	importPath, err := cli.Prompt("import path/module", goModule)
```

Comment:
Hm, why does this have to be interactive. I don't see a reason why the user would NOT want to use the one in `go.mod`. Maybe only ask this question, if a module cannot be detected?

Reply:
Go point